tools/amd_build/build_amd.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
```

Comment:
We have been putting some assorted tools in the `tensorpipe/misc` folder. It would be good to use a single directory for all these.

Reply:
In the process of moving this file into hipify-torch repo, which is added a git submodule. 
https://github.com/ROCmSoftwarePlatform/hipify-torch.git
Once the PRs into hipify-torch is merged, I will update this PR.